content-src/lib/shim.js

```diff
@@ -1,7 +1,48 @@
 const fakeFrecent = [
   {
     title: "Facebook",
-    leadImage: "https://www.facebook.com/images/fb_icon_325x325.png",
```

Comment:
yeah i think it was just like, me not using the same schema. 

In the future i think this will need to change anyway, we will need a way to specify different resolutions for images, favicons, etc. Keeping it `image` is probably OK for now but we'll need to update things once we lock that down. Maybe let's discuss this week